scripts/c2r_script.py

```diff
@@ -8,6 +8,12 @@
 
 from urllib2 import urlopen
 
+# SCRIPT_TYPE is either 'CONVERSION' or 'ANALYSIS'
+# Value is set in signed yaml envelope in content_vars (CONVERT2RHEL_SCRIPT_TYPE)
+SCRIPT_TYPE = os.getenv("RHC_WORKER_CONVERT2RHEL_SCRIPT_TYPE", None)
+IS_CONVERSION = SCRIPT_TYPE == "CONVERSION"
+IS_ANALYSIS = SCRIPT_TYPE == "ANALYSIS"
```

Comment:
```suggestion
SCRIPT_TYPE = os.getenv("RHC_WORKER_CONVERT2RHEL_SCRIPT_TYPE", "").upper()
IS_CONVERSION = SCRIPT_TYPE == "CONVERSION"
IS_ANALYSIS = SCRIPT_TYPE == "ANALYSIS"
```

Reply:
Enforcing that we will always get a string and whatever comes in it will be upper case.

Reply:
```python
>>> "ConVERSION" == "CONVERSION"
False
```